javascript/builtins/Promise.json

```diff
@@ -249,7 +249,8 @@
                 "version_added": "50"
               },
               "safari": {
-                "version_added": false
+                "version_added": true,
+                "version_removed": "8"
```

Comment:
Promise.finally is relatively new. I don't think it was in Safari before 8 and removed in 8 also seems off.